src/Helper/WebformHelper.php

```diff
@@ -68,4 +96,93 @@ public function getTemplateContext(WebformSubmissionInterface $webformSubmission
     ];
   }
 
+  /**
+   * Send digital post.
+   *
+   * @param string $submissionId
+   *   The submission ID.
+   * @param array $handlerConfiguration
+   *   Handler config.
+   *
+   * @throws CprElementNotFoundInSubmissionException
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \ItkDev\Serviceplatformen\Service\Exception\ServiceException
+   */
+  public function sendDigitalPost(string $submissionId, array $handlerConfiguration) {
+    /** @var \Drupal\webform\Entity\WebformSubmission $submission */
+    $webform_submission = $this->getSubmission($submissionId);
+
+    $submissionData = $webform_submission->getData();
+
+    if (!array_key_exists($handlerConfiguration['cpr_element'], $submissionData)) {
+      $this->getLogger()->error(
+        'The chosen CPR element not found in submission!'
+      );
+
+      throw new CprElementNotFoundInSubmissionException();
+    }
+
+    /** @var \Drupal\os2forms_cpr_lookup\CPR\CprServiceResult $cprSearchResult */
+    $cprSearchResult = $this->cprService->search($submissionData[$handlerConfiguration['cpr_element']]);
+
+    $context = $this->getTemplateContext($webform_submission, $cprSearchResult, $handlerConfiguration);
+
+    if (TRUE === $handlerConfiguration['debug']) {
+      $this->templateManager->renderPdf($handlerConfiguration['template'], $context, TRUE);
+      return;
+    }
```

Comment:
This does not make sense when processing a queue job. The “debug” configuration was meant to render the pdf when the form is submitted and display it to the user. We should remove this way of debugging the pdf or still do it on form submission (rather than adding a job to the queue).

Reply:
Debugging lines removed. We still have drush command available for testing.